### PR TITLE
Scrub the type value from release_payload.

### PR DESCRIPTION
**Pull request by ralphbean**
Hoping that @lubomir can provide some insight here.

### CODE REVIEW

File: pdcupdater/utils.py [hunk 1]
```diff
@@ -92,6 +92,14 @@ def ensure_release_exists(pdc, release_id, release):
         release_payload.update(dict(
             active=True,
         ))
+
+        # XXX - Both release_type and type should have the same value, but PDC
+        # doesn't expect that the 'type' value should be present.  If it is
+        # present, we fail with a 400 error.  Here, we delete it until the
+        # pungi/PDC authors tell us otherwise.
+        if 'release_type' in release_payload and 'type' in release_payload:
+            del release_payload['type']
```
Comment (lubomir) [inline pdcupdater/utils.py]: Where is the `release_payload` coming from? PDC should only return `release_type`, and if it's extracted from parsed `composeinfo.json`, it should be `type`.

Comment (ralphbean) [inline pdcupdater/utils.py]: Ah, it comes from [here](https://kojipkgs.fedoraproject.org/compose/Fedora_Docker-25-20170109.0/compose/metadata/composeinfo.json), `composeinfo['payload']['release']`, but then for reasons I don't remember I add `release_payload['release_type'] = 'ga'`, hardcoded, elsewhere in my code.

Comment (ralphbean) [inline pdcupdater/utils.py]: OK, thinking a little more clearly now.

I think we just need to update our PDC instance in Fedora first, and then sort out if this is still a problem.  We should almost certainly drop our `release_type` field and use just the `type` field instead.

Comment (ralphbean) [inline pdcupdater/utils.py]: After reading your email, it's even more clear.  #39 should have the answer.

**Pull request closed without merging**